llvm/lib/Transforms/Scalar/LowerMatrixIntrinsics.cpp

```diff
@@ -1138,8 +1135,36 @@ class LowerMatrixIntrinsics {
       LowerColumnMajorStore(Inst);
       break;
     case Intrinsic::abs:
-    case Intrinsic::fabs:
-      return VisitUniformIntrinsic(cast<IntrinsicInst>(Inst));
+    case Intrinsic::fabs: {
+      IRBuilder<> Builder(Inst);
+
+      MatrixTy Result;
+
+      MatrixTy M = getMatrix(Inst->getOperand(0), Shape, Builder);
+
+      Builder.setFastMathFlags(getFastMathFlags(Inst));
```

Comment:
```suggestion
      IRBuilder<> Builder(Inst);
      MatrixTy Result;
      MatrixTy M = getMatrix(Inst->getOperand(0), Shape, Builder);
      Builder.setFastMathFlags(getFastMathFlags(Inst));
```

not sure if we need newlines between all of those